### ISSUE TITLE
Bug: Song Lists locked - Version: 1.5.22 unstable Built: Jan  4 2019 19:10:01 Win64

### ISSUE THREAD

**Issue by jpoconnor61** [OpenKJ/OpenKJ#138]
Behaviour also noted on .21 version

Conditions are with a list of singers, multiple songs loaded for some; Karaoke Autoplay enabled; Break Music enabled to play between songs.

After a few songs have been sung, it becomes impossible to change singer order using drag and drop; similarly the song lists for each singer cannot be re-ordered.

Re-start required to resolve.

Some attempts to change ordering are signalled with a "no-entry" tool-tip symbol , but not always. On a few occasions the OpenKJ software will stop when making the attempt.

Minor issue, probably unrelated - Singer list usually does not show next for for each entry, but I did see it working briefly after a KaraokeDB update,

JOC

**Comment by coyote1357** [comment]
Thanks for the report, I'll look into it.

**Comment by coyote1357** [comment]
This should have been fixed a few versions ago in unstable, but I forgot to update the ticket, sorry about that!

**Comment by coyote1357** [comment]
Feel free to re-open if it's still a problem, I'm going to go ahead and close for now.

**Comment by jpoconnor61** [comment]
As it happens, I haven't been using the software for a few months, having
lost my regular gig.

I did spot some of the announcements addressing this problem though.

I have installed latest unstable, and been lightly testing, and am still
seeing some of this behaviour - specifically the inability to re-order the
singers, sometimes the interface does not respond, sometimes I get the
'no-entry' tooltip, and sometime a complete halt of OpenKJ.

I still don't have any sense of what's different from instances where it
works just fine.

I have the logging turned on, so can supply some of those if it will be of
any help to you.

JOC
